### ISSUE TITLE
Add list of wallets known to support NWC, and compatible with Damus

### ISSUE THREAD

**Issue by alltheseas** [damus-io/damus#1976]
_what happens_

Some Damus users are not sure which wallets support NWC. User tried to add WOS, which currently does not support NWC.

https://damus.io/note1rmgk6ahj34msj8pukcqvfejtqwp84hk7sqtlvrmnzdcy4hceurzs5lzxal

> Having difficulty adding a wallet in latest damus test flight update. When I select attach wallet - getting error below without inputting anything. 

> Trying to add a WoS. I don’t even get far enough to paste my address. Selecting “attach wallet” opens the input screen but immediately bounces me back to error photo. (Thanks for reply)

_suggestion_

Provide listed of supported, and tested NWC wallets.

**Comment by ericholguin** [comment]
Do we know which ones currently support it? Alby, Mutiny, and who else?

**Comment by alltheseas** [comment]
Yes, start with those two. I heard Fedi also supports, and have not seen evidence of this or verified

**Comment by alltheseas** [comment]
@ericholguin https://nwc.dev/

**Comment by alltheseas** [comment]
see https://github.com/nostrability/nostrability/issues/77

**Comment by alltheseas** [comment]
Closing as part of a backlog triage — we're narrowing focus to the most actionable issues to reduce noise for contributors.

If this is still relevant, please reopen with updated context or reproduction steps. Thanks for reporting!
